content/birds/sondra.md

```diff
@@ -14,15 +14,17 @@ social:
   dribbble: sondraeby
 summary: |
   Sondra
-  is a brand and interface designer --
+  is a brand and UX/UI designer --
   as well as a marketing expert,
-  and usability tester.
+   usability tester, and OOUX strategist.
```

Comment:
Should we consider spelling out UI & UX, or adding an abbreviation tag around OOUX with the full title? Just thoughts, not a blocker.

Reply:
I'd go along with spelling it out the first time it is used.

Reply:
I think so, yes.